desktop/src/gui/menu_bar.rs

```diff
@@ -64,7 +66,14 @@ impl MenuBar {
                 player.set_is_playing(!player.is_playing());
             }
         }
-        if egui_ctx.input_mut(|input| input.consume_shortcut(&Self::SHORTCUT_FULLSCREEN)) {
+        let mut fullscreen_pressed =
+            egui_ctx.input_mut(|input| input.consume_shortcut(&Self::SHORTCUT_FULLSCREEN));
+        if cfg!(windows) && !fullscreen_pressed {
+            // TODO We can remove this shortcut when we add some kind of preferences.
+            fullscreen_pressed = egui_ctx
+                .input_mut(|input| input.consume_shortcut(&Self::SHORTCUT_FULLSCREEN_WINDOWS));
```

Comment:
It irks me a little that the second shortcut is only consumed if the first one hasn't been (instead of consuming both unconditionally and combining the results after), but it's such a minor thing that is very unlikely to matter, that it's not worth blocking for.